.orchestra/support/revng-container/Dockerfile

```diff
@@ -0,0 +1,46 @@
+FROM docker.io/library/ubuntu:22.04 as BASE
+
+ENV DEBIAN_FRONTEND=noninteractive
+
+COPY apt.no-extra /etc/apt/apt.conf.d/no-extra
+RUN apt update && \
+    apt install -y \
+        # Needed for recompilation pipe (-dev included due to linking problems)
+        libc6 libc6-dev \
+        # Needed for revng.cli etc.
+        python3 ca-certificates \
+        # some LLVM libs are not searched properly
+        libllvm12 libc++1-12 libatomic1 && \
```

Comment:
Be more specific. These are all bugs, we need to at least have a note on how to fix this.

Reply:
Ditto wrt the review comment above